### PR TITLE
CI: use apt instead of dnf

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
because it's an ubuntu based image instead of a fedora one

### CODE REVIEW

File: .github/workflows/docs.yml
Comment (GuillaumeGomez) [inline .github/workflows/docs.yml:41]: No `sudo`? Also: didn't pay attention to that...

Comment (bilelmoussaoui) [reply .github/workflows/docs.yml:41]: Somehow the ubuntu image doesn't require one because you're sudoer by default

**Review by sdroege — APPROVED** [review]

File: .github/workflows/docs.yml
Comment (sdroege) [reply .github/workflows/docs.yml:41]: Also `apt` instead of `apt-get` is the modern way, but doesn't make a difference really

**Comment by bilelmoussaoui** [comment]
do not merge until #573 is merged and the image got updated

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**
